src/commands/classified/pipeline.rs

```diff
@@ -48,26 +46,5 @@ pub(crate) async fn run_pipeline(
         };
     }
 
-    use futures::stream::TryStreamExt;
-    if let Some(input) = input {
-        let mut output_stream: OutputStream = input.into();
-        loop {
-            match output_stream.try_next().await {
```

Comment:
Is the intent for this removal that commands should be handling Ctrl-C?

Reply:
What I'm doing here probably need more explanation than I gave it.  Basically we no longer will drain the pipeline. When we `run_pipeline`, what comes out is a pipeline that's still lazy and ready to be processed.

This puts the requirement to process ctrl-c (and the stream itself) in other places in the code.

eg) 
https://github.com/nushell/nushell/pull/1359/files/f549c755cd92f84b5b04f7e675c355be292e4c9b#diff-dba02791b4baf8a328939029ac920005R654